scripts/custom-tile.sh

```diff
@@ -50,9 +65,20 @@ ENV VERSION="${VERSION}" GOPATH="${TMP_GOPATH}" PATH="${PATH}:${TMP_GOPATH}/bin"
 COPY . .
 RUN scripts/build-custom-tile-docker.sh
 __DOCKERFILE__
+
+  # remove previous custom tiles
+  rm -f stackdriver-nozzle-custom*
+
+  # remove previous tile containers and image (safely)
```

Comment:
Is this step necessary if it's being run in a container?

Reply:
I just updated these scripts. Most of this work was duplicated by the Makefile. The docker image now uses the Makefile